oci.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import logging
+
+from oci import config
+from oci import gerrit
+from oci import jenkins
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='A command line tool for working with Ovirt CI')
+
+    parser.add_argument(
+        '--verbose',
+        help="increase output verbosity",
+        action="store_true")
+    subparsers = parser.add_subparsers(title="commands")
+
+    build_artifacts = subparsers.add_parser(
+        "build-artifacts",
+        help="start the build-artifacts stage")
+    build_artifacts.set_defaults(command=run_build_artifacts)
```

Comment:
This will be nicer if the function name is build_artifacts.

Reply:
Done.